aea/helpers/base.py

```diff
@@ -197,12 +197,12 @@ def send_control_c(
     if platform.system() == "Windows":
         if process.stdin:  # cause ctrl-c event will be handled with stdin
             process.stdin.close()
-        os.kill(process.pid, signal.CTRL_C_EVENT)  # type: ignore   # pylint: disable=no-member
+        process.send_signal(signal.CTRL_C_EVENT)  # type: ignore # pylint: disable=no-member
```

Comment:
we could send `CTRL_BREAK_EVENT` instead, which cannot be redirected, but it would no longer align with the function name.